### ISSUE TITLE
MuseScore 3.0.0 Release

### ISSUE THREAD

**Issue by ovari** [flathub/org.musescore.MuseScore#17]
https://musescore.org/en/3.0

**Comment by ovari** [comment]
https://musescore.org/en/3.0#comment-880841

**Comment by ovari** [comment]
The MuseScore icon in the Linux Mint Cinnamon menu still shows the MuseScore 2 icon. Has the Flatpak been updated with the new icon or is this an issue with [Linux Mint](https://linuxmint.com/)?

If this is an issue with Linux Mint, does an issue need to be created at [the cinnamon-menu library for Linux Mint](https://github.com/linuxmint/cinnamon-menus) Thank you 

![MuseScore 3 shows the MuseScore 2 icon in the Linux Mint menu](https://user-images.githubusercontent.com/17465872/50426517-e7bc8180-08e3-11e9-8bb0-b2b10151f5c7.png)

**Comment by jurf** [comment]
@ovari you are using a custom icon pack. It needs to be updated there.

**Comment by jurf** [comment]
@ovari yeah, JACK currently isn’t included. I’ll try asking if it is possible to hack around somehow, but it’s a gap in Flatpak intended for PipeWire to fill.

**Comment by eszlari** [comment]
There are already some apps on Flathub that include JACK support, just use the search function.

**Comment by jurf** [comment]
@eszlari does jack actually work there though? Isn’t it just needed for them to build?

**Comment by eszlari** [comment]
Yes: https://github.com/flathub/org.shotcut.Shotcut/issues/4
